spec/controllers/automated_tests_controller_spec.rb

```diff
@@ -8,14 +8,43 @@
       # TODO: write tests
     end
     context 'GET manage' do
-      before :each do
-        get_as user, :manage, params: params
-      end
-      it 'User should be able to view the Automated Testing manage page' do
-        expect(response.status).to eq(200)
+      context 'Manage page' do
+        before :each do
+          get_as user, :manage, params: params
+        end
+        it 'User should be able to view the Automated Testing manage page' do
+          expect(response.status).to eq(200)
+        end
+        it 'should render the assignment_content layout' do
+          expect(response).to render_template('layouts/assignment_content')
+        end
       end
-      it 'should render the assignment_content layout' do
-        expect(response).to render_template('layouts/assignment_content')
+      context 'Getting the test student grouping' do
+        context 'When the assignment already has test grouping' do
+          let(:test_student) { create(:test_student, user_name: User::TEST_STUDENT_USER_NAME) }
+          let(:group) { create(:group, group_name: 'test_student_group') }
+          let(:grouping) { create(:grouping, group: group, assignment: assignment) }
+          let!(:membership) { create(:inviter_student_membership, user: test_student, grouping: grouping) }
+          it 'should return the existing test grouping' do
+            get_as user, :manage, params: params
+            expect(assigns(:test_grouping)).to eq(grouping)
+            expect(assigns(:test_grouping).memberships.first).to eq(membership)
+          end
+        end
+        context 'When there is no test grouping exists for the assignment' do
+          before :each do
+            get_as user, :manage, params: params
+          end
+          it 'should return a newly created test grouping' do
+            expect(assigns(:test_grouping)).to be_truthy
```

Comment:
`be_truthy` is too broad of a check and I'm pretty sure will always be true even if an error occurs.